src/pages/index.js

```diff
@@ -27,44 +31,165 @@ function openPreviewModal(data) {
 
 /* Instantiations*/
 
-const cardSection = new Section(
-  { items: initialCards, renderer: renderCard },
-  ".cards__gallery"
-);
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d5d1d874-a161-48bd-b3b4-ae8e38400ed9",
+    "Content-Type": "application/json",
+  },
+});
 
+let cardSection;
+
+api
+  .getUserInfo()
+  .then((res) => {
+    userInfo.setUserInfo(res);
+    userInfo.setUserAvatar(res);
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+
+api
+  .getInitialCards()
+  .then((res) => {
+    cardSection = new Section(
+      { items: res, renderer: renderCard },
+      ".cards__gallery"
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.log(err);
+  });
 function renderCard(cardData) {
   const card = new Card(
     cardData,
     selectors.cardTemplate,
-    openPreviewModal
+    openPreviewModal,
+    handleCardDeleteClick,
+    handleLikeClick
   ).getCardElement();
   cardSection.addItem(card);
 }
 
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+const userInfo = new UserInfo(".profile__title", ".profile__description", avatarImage);
 const profileEditForm = new PopupWithForm(
   "#profile-edit-modal",
   handleProfileEditSubmit
 );
 const cardEditForm = new PopupWithForm("#add-card-modal", handleCardFormSubmit);
 const previewImagePopup = new PopupWithImage("#modal__preview-card");
+const confirmDeletePopup = new PopupWithConfirm("#delete-card-modal");
+
+const avatarImagePopup = new PopupWithForm(
+  "#edit-avatar-modal",
+  handleAvatarFormSubmit
+);
 
 /*Validation*/
 
 const editFormValidator = new FormValidator(config, profileForm);
 const addCardValidator = new FormValidator(config, cardForm);
+const deleteCardValidator = new FormValidator(config, deleteCardForm);
```

Comment:
`deleteCardValidator`  is not needed because it doesn' thave any inputs for validating